superset/views/api.py

```diff
@@ -40,7 +40,15 @@
 if TYPE_CHECKING:
     from superset.common.query_context_factory import QueryContextFactory
 
-get_time_range_schema = {"type": "string"}
+get_time_range_schema = {
+    "type": ["string", "array"],
+    "items": {
+        "type": "object",
+        "properties": {
+            "timeRange": {"type": "string"},
```

Comment:
Let's use flexible data structure in the object so that support more config in time range request, e.g. timezone. Our forked version already supported timezone in time range, will open separated PR.